x-pack/plugins/security_solution/server/lib/detection_engine/rule_execution_log/rule_execution_log_client.ts

```diff
@@ -20,17 +20,20 @@ import {
 } from './types';
 
 export interface RuleExecutionLogClientArgs {
+  isRuleRegistryEnabled: boolean;
   ruleDataService: IRuleDataPluginService;
   savedObjectsClient: SavedObjectsClientContract;
 }
 
-const RULE_REGISTRY_LOG_ENABLED = false;
-
 export class RuleExecutionLogClient implements IRuleExecutionLogClient {
   private client: IRuleExecutionLogClient;
 
-  constructor({ ruleDataService, savedObjectsClient }: RuleExecutionLogClientArgs) {
-    if (RULE_REGISTRY_LOG_ENABLED) {
+  constructor({
```

Comment:
This will probably be superceded by Dmitrii's changes that were recently merged in https://github.com/elastic/kibana/pull/112286/files